cclib/parser/molcasparser.py

```diff
@@ -616,6 +616,57 @@ def extract(self, inputfile, line):
 
             self.atomcoords.append(atomcoords)
 
+        #        All orbitals with orbital energies smaller than  E(LUMO)+0.5 are printed
+        #
+        #  ++    Molecular orbitals:
+        #        -------------------
+        #
+        #        Title: RKS-DFT orbitals
+        #
+        #        Molecular orbitals for symmetry species 1: a
+        #
+        #            Orbital        1         2         3         4         5         6         7         8         9        10
+        #            Energy      -10.0179  -10.0179  -10.0075  -10.0075  -10.0066  -10.0066  -10.0056  -10.0055   -9.9919   -9.9919
```

Comment:
Is there ever a case where the energy will have additional digits?

Reply:
Don't know about that. Let me see if the other programs give similar values for the energy.